### ISSUE TITLE
Condition for Showing "YGM cloned cereal dependency" in CMake

### ISSUE THREAD

**Issue by KIwabuchi** [LLNL/ygm#39]
This is a super minor bug fix regarding the line below:
https://github.com/LLNL/ygm/blob/master/CMakeLists.txt#L77

As find_package cannot find packages downloaded by FetchContent in a parent project that depends on YGM, we need to put another logic there to prevent showing "YGM cloned cereal dependency."
I think what we need to do is something like:

```cmake
    # Stop using FetchContent_MakeAvailable
    # FetchContent_MakeAvailable(cereal)

    FetchContent_GetProperties(cereal)
    if (cereal_POPULATED)
        message(STATUS "YGM found already populated cereal dependency: " ${cereal_SOURCE_DIR})
    else ()
        set(JUST_INSTALL_CEREAL ON)
        FetchContent_Populate(cereal)
        message(STATUS "YGM cloned cereal dependency: " ${cereal_SOURCE_DIR})
    endif ()
```

As you can see, it may be too complicated code to show the message.
So, just removing the message might be better?

**Comment by rogerpearce** [comment]
@KIwabuchi @bwpriest 

Is this still an issue?

**Comment by bwpriest** [comment]
The pattern that I am using in libraries that depend on ygm uses a nested conditional such that if ygm is not found, i. e. it is not installed, we clone it using FetchContent which also clones and makes cereal available to the dependent project (because it runs ygm's CMakeLists.txt), in which case no further work is required. The dependent project assumes that if ygm is installed and findable, then so is cereal. See my pull request for SaltAtlas or the current state of croquis for an example. I am sure that there are other ways to do this workflow, though. Keita's idea might be more disciplined.

**Comment by bwpriest** [comment]
I believe that this issue is addressed in PR #62

**Comment by steiltre** [comment]
Merged PR [#62](https://github.com/LLNL/ygm/pull/62)
